relay-event-normalization/src/normalize/processor.rs

```diff
@@ -199,6 +207,22 @@ impl<'a> Processor for NormalizeProcessor<'a> {
 
         Ok(())
     }
+
+    fn process_span(
+        &mut self,
+        span: &mut Span,
+        _: &mut Meta,
+        state: &ProcessingState<'_>,
+    ) -> ProcessingResult {
+        validate_span_timestamps(span)?;
+        validate_span_ids(span)?;
```

Comment:
I'm not convince that splitting this function into helper functions is the right thing to do for this PR. It's small enough that I would keep as it was.